sourcegraph/add-sourcegraph-extension.batch.yml

```diff
@@ -0,0 +1,20 @@
+name: add-sourcegraph-extension
+description: Add the Sourcegraph extension as a recommended vscode extension
+on:
+  - repositoriesMatchingQuery: file:.vscode/settings.json
```

Comment:
doesn't this kind of config go into `.vscode/extensions.json`?

Reply:
iirc, it can also be set in here.

Might be worth checking if there's an `extensions.json` file in the python script though, to avoid duplication.

Reply:
Yeah, Erik is right. I think it needs to be in `extensions.json`.

Check [here](https://code.visualstudio.com/docs/editor/extension-marketplace#_workspace-recommended-extensions).

According to the [schema](https://code.visualstudio.com/docs/getstarted/settings#_settingsjson) for `settings.json`, I don't see a field for `recommendations`